pkg/monitoring/metrics/virt-controller/vmistats_collector.go

```diff
@@ -250,30 +251,32 @@ func getVMIPod(vmi *k6tv1.VirtualMachineInstance) string {
 
 func getVMIInstancetype(vmi *k6tv1.VirtualMachineInstance) string {
 	if instancetypeName, ok := vmi.Annotations[k6tv1.InstancetypeAnnotation]; ok {
-		return fetchResourceName(instancetypeName, instancetypeMethods.InstancetypeStore)
+		key := fmt.Sprintf("%v/%v", vmi.Namespace, instancetypeName)
```

Comment:
nit - I like to use [`NamespacedName`](https://pkg.go.dev/k8s.io/apimachinery/pkg/types#NamespacedName) instead of `Sprintf` for this.

```
key := types.NamespacedName{
  Namespace: vmi.Namespace,
  Name: instancetypeName,
}
return fetchResourceName(key.String(), instancetypeMethods.InstancetypeStore)
```

Reply:
updated